src/themes/teams/siteVariables.ts

```diff
@@ -50,11 +50,13 @@ export const shadowLevel1 = '0 .2rem .4rem -.075rem rgba(0, 0, 0, 0.1)'
 // FONT SIZES
 //
 export const fontSizes = {
+  smallest: pxToRem(8),
   smaller: pxToRem(10),
   small: pxToRem(12),
   medium: pxToRem(14),
   large: pxToRem(18),
   larger: pxToRem(24),
+  largest: pxToRem(28),
```

Comment:
I'm not sure that it's the best idea, should I revert this change?

Reply:
Reverted: https://github.com/stardust-ui/react/issues/136#issuecomment-447152727